File: core-java-practice/gcr-codebase/programming-elements/SimpleInterest.java
import java.util.Scanner;

public class SimpleInterest {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        //take input from user
        double pri = sc.nextDouble();
        double rate = sc.nextDouble();
        double time = sc.nextDouble();
        double si = (pri * rate * time) / 100;//cal simple interset
        System.out.print(si);

    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/SimpleInterest.cs

class SimpleInterest {
    static void Main(String[] args) {
        //take input from user
        double pri = Convert.ToDouble(Console.ReadLine());
        double rate = Convert.ToDouble(Console.ReadLine());
        double time = Convert.ToDouble(Console.ReadLine());
        double si = (pri * rate * time) / 100;//cal simple interset
        Console.WriteLine(si);

    }
}
